letter-sender/src/main/java/com/christmas/letter/sender/exceptions/LetterControllerAdvice.java

```diff
@@ -0,0 +1,26 @@
+package com.christmas.letter.sender.exceptions;
+
+import java.util.Date;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class LetterControllerAdvice extends ResponseEntityExceptionHandler {
```

Comment:
Let's give this component a better name, I wouldn't use Advice as that is only an implementation detail.

Reply:
name updated to Advice

Reply:
This is still pending.